homeassistant/components/binary_sensor/verisure.py

```diff
@@ -54,6 +54,7 @@ def available(self):
             "$.doorWindow.doorWindowDevice[?(@.deviceLabel=='%s')]",
             self._device_label) is not None
 
-    def update(self):
+    @staticmethod
+    def update():
```

Comment:
I'm not a fan of changing these to staticmethod as it changes the method signature of the one that it overwrites. Instead, verisure should not use a global so that we refer to the hub as `self.hass.data[DATE_VERISURE_HUB]`